README.macOS

```diff
@@ -20,3 +20,12 @@ Building BZFlag from Source
 ---------------------------
 BZFlag builds on macOS are now done using premake5 and Xcode 7 (or later).
 Please refer to the README.premake5 file for build instructions.
+
+
+Known Issues
+------------
+The following warnings can be expected in a MacOS build (for now):
```

Comment:
From a context of historical tendencies in our code/documentation, I would strongly prefer that we do not document issues in this manner. Generally speaking, such comments tend to be forgotten about, the context is lost, and we end up with TODOs and FIXMEs littered throughout the code. Regarding READMEs specifically, I would prefer that they generally be set-it-and-forget-it, rather than having some kind of working list of issues or deficiencies. I am also working on resolving these last two warnings, after which our code should compile cleanly on macOS.

Reply:
Again, I don’t mind reverting this file entirely. My purpose in adding this section was to inform a Mac developer of “expected” warnings so they would know they hadn’t done anything wrong.